cpg_infra/billing_aggregator/driver.py

```diff
@@ -181,21 +262,30 @@ def setup_aggregator_functions(self):
             if function == 'seqr':
                 # 2GB is not enough for seqr
                 memory = '2560M'
+
+            # Create the source archive
+            source_archive = self.create_source_archive(
+                'billing-aggregator-source-code',
```

Comment:
Wonder if this should be `f'billing-aggregator-{function}-source-code'`

Reply:
Ah good call. I don't think so, it get's used by all the aggregator functions. But I have moved it out of the function loop for clarity